package.json

```diff
@@ -38,9 +38,34 @@
       "npm run server:lint"
     ]
   },
+  "eslintConfig": {
+    "plugins": [
+      "prettier"
+    ],
+    "extends": [
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "no-throw-literal": "off",
```

Comment:
I think they should be removed from here, because they only make sense on the server part.

Reply:
oops forget this one